sdk/identity/azure-identity/pom.xml

```diff
@@ -32,7 +32,7 @@
     <dependency>
       <groupId>com.microsoft.azure</groupId>
       <artifactId>msal4j</artifactId>
-      <version>1.6.2</version> <!-- {x-version-update;com.microsoft.azure:msal4j;external_dependency} -->
+      <version>1.7.0</version> <!-- {x-version-update;com.microsoft.azure:msal4j;external_dependency} -->
```

Comment:
NOTE: I'll update the external_dep file before merging, as that will impact other libraries using MSAL too.